navigator/runtime-compose/api/navigator-runtime-compose.api

```diff
@@ -21,7 +25,9 @@ public final class com/freeletics/mad/navigator/compose/NavigationSetupKt {
 	public static final fun NavigationSetup (Lcom/freeletics/mad/navigator/NavEventNavigator;Landroidx/compose/runtime/Composer;I)V
 }
 
-public final class com/freeletics/mad/navigator/compose/ScreenDestination : com/freeletics/mad/navigator/compose/NavDestination {
+public final class com/freeletics/mad/navigator/compose/ScreenDestination : com/freeletics/mad/navigator/compose/ContentDestination {
```

Comment:
These destinations by being public expose internal `ContentDestination`

Reply:
Yes, the reason why this works and is ok is that these subclasses are also internal but use `@PublishedApi`, so from a binary perspective this is public but from a source perspective all of this is internal.